backend/lunchmenuloader/homeOfficeLoader.coffee

```diff
@@ -0,0 +1,44 @@
+
+LunchmenuLoader = require './lunchmenuLoader'
+
+
+class HomeOfficeLoader extends LunchmenuLoader
+    constructor: () ->
+        @name = 'Home Office'
+        @homepage = 'http://homeofficebistro.cz/'
+        @downloadUrl = 'http://homeofficebistro.cz/tydenni-menu/'
+        @phoneNumber = '+420 778 767 768'
+        @address =
+            street: 'Radlická 3185/1C'
+            city: 'Praha 5'
+            zip: 15000
+            map:
+                lat: 50.0716481
+                lng: 14.4011792
+
+    parse: (restaurant, $) ->
+        days = ['Ponděl�', 'Úterý', 'Středa', 'Čtvrtek', 'Pátek', 'Sobota', 'Neděle']
```

Comment:
To mám z toho, že občas používám vim :) Kopíroval jsem to z ilNostry tuším a asi tam je rozbité kódování.